### PR TITLE
added category, website, bugs, repository

### PR DESCRIPTION
**Pull request by janklostermann**

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -8,9 +8,14 @@
 	<dependencies>
 		<owncloud min-version="9.0" max-version="9.1" />
 	</dependencies>
+
```
Comment (georgehrke) [inline appinfo/info.xml:11]: Why did you add this empty line?

File: appinfo/info.xml [hunk 2]
```diff
@@ -8,9 +8,14 @@
...
 	<description>Calendar GUI for ownCloud's integrated CalDAV server</description>
 	<documentation>
 		<user>http://doc.owncloud.org/server/9.0/user_manual/pim/calendar.html</user>
 	</documentation>
+	<category>productivity</category>
```
Comment (georgehrke) [inline appinfo/info.xml:16]: Are this properties documented somewhere?
I've never seem them in other apps or anywhere in core.

File: appinfo/info.xml (hunk 1 above)
Comment (janklostermann) [inline appinfo/info.xml:11]: Had no intention. You can kick it.

Am 2016-06-02 um 18:24 schrieb Georg Ehrke:

> In appinfo/info.xml 
> https://github.com/owncloud/calendar/pull/564#discussion_r65572419:
> 
> > @@ -8,9 +8,14 @@
> >     <dependencies>
> >         <owncloud min-version="9.0" max-version="9.1" />
> >     </dependencies>
> > +
> 
> Why did you add this empty line?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> https://github.com/owncloud/calendar/pull/564/files/3f50ff3217d392ced984e6d5dc00210a056b3d13#r65572419, 
> or mute the thread 
> https://github.com/notifications/unsubscribe/ALEnw9GxCIS89_xxeJ2-gdxXmq11lNnnks5qHwPIgaJpZM4Isqi9.

## 

Jan Klostermann
- The Agile Management Sounding-Board -
  Systems Engineer & Mediator
  Am Isarkanal 24/15
  81379 München

Tel.: +49-89/74563382
Mobil: +49-179/4628819
Sip: +49-89/21551663 sip:08921551663@sipgate.de
Skype: jan.klostermann callto://jan.klostermann/
Threema: P58VSJUY threema://add?id=P58VSJUY
Email: jk@janklostermann.de mailto:jk@janklostermann.de
Netzwerke: Xing https://www.xing.com/profile/Jan_Klostermann, LinkedIn 
http://de.linkedin.com/in/janklostermann

File: appinfo/info.xml (hunk 2 above)
Comment (janklostermann) [inline appinfo/info.xml:16]: https://doc.owncloud.org/server/9.0/developer_manual/app/info.html

But you don't seem to be the only one who has never seen them ;-)

Am 2016-06-02 um 18:26 schrieb Georg Ehrke:

> In appinfo/info.xml 
> https://github.com/owncloud/calendar/pull/564#discussion_r65572627:
> 
> > ```
> > <description>Calendar GUI for ownCloud's integrated CalDAV server</description>
> > <documentation>
> >     <user>http://doc.owncloud.org/server/9.0/user_manual/pim/calendar.html</user>
> > </documentation>
> > ```
> > -   <category>productivity</category>
> 
> Are this properties documented somewhere?
> I've never seem them in other apps or anywhere in core.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> https://github.com/owncloud/calendar/pull/564/files/3f50ff3217d392ced984e6d5dc00210a056b3d13#r65572627, 
> or mute the thread 
> https://github.com/notifications/unsubscribe/ALEnw1QvdXMSgNYFT4U6-XgqGJ2vN012ks5qHwQagaJpZM4Isqi9.

## 

Jan Klostermann
- The Agile Management Sounding-Board -
  Systems Engineer & Mediator
  Am Isarkanal 24/15
  81379 München

Tel.: +49-89/74563382
Mobil: +49-179/4628819
Sip: +49-89/21551663 sip:08921551663@sipgate.de
Skype: jan.klostermann callto://jan.klostermann/
Threema: P58VSJUY threema://add?id=P58VSJUY
Email: jk@janklostermann.de mailto:jk@janklostermann.de
Netzwerke: Xing https://www.xing.com/profile/Jan_Klostermann, LinkedIn 
http://de.linkedin.com/in/janklostermann

Comment (georgehrke) [inline appinfo/info.xml:16]: hehe :)

File: appinfo/info.xml (hunk 1 above)
Comment (georgehrke) [inline appinfo/info.xml:11]: Will do and merge afterwords.

Thanks a lot for your contribution! :)

**Comment by georgehrke** [comment]
Removed empty line and merged manually
https://github.com/owncloud/calendar/commit/c984ca4fc4e5e962ec7ebea6fcc4409cc5c20751

**Comment by georgehrke** [comment]
Thanks for your contribution! :)
